src/components/PaymentInformation/PaymentMethod.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Text from '../Text';
+import SectionHeader from '../SectionHeader';
+import Wrapper from './Wrapper';
+import PaymentCard from './PaymentCard';
+import CardName from './CardName';
+import Price from './Price';
+
+function PaymentMethod({ cardNumber, expiries, price }) {
+  const hideCardNumber = () => `XXXX-XXXX-XXXX-${cardNumber.slice(12)}`;
```

Comment:
Would hoist it above component and provide cardNumber as arg:
```
function hiddenCardNumber(cardNumber) {
  return XXX...
}
```
It will make it easily testable.